scripts/check_urls/check_urls.ts

```diff
@@ -0,0 +1,57 @@
+import fs from "fs";
+import YAML from "yaml";
+
+interface UrlObject {
+  description: string;
+  tags: string[];
+  title: string;
+  url: string;
+}
+const filePath = "../../data/resources.yml";
+
+fs.readFile(filePath, "utf-8", (err, data) => {
+  const urls: UrlObject[] = YAML.parse(data);
+
+  const checkUrl = async (url: UrlObject) => {
+    try {
+      const response = await fetch(new URL(url.url).toString(), {
+        method: "HEAD",
+      });
```

Comment:
```suggestion
      const response = await fetch(new URL(url.url).toString(), {
        method: "HEAD",
        redirect: "manual",
      });
```
would mean that redirects would give response code `301`, and you don't need to look at `.redirected`.

Reply:
Actually, I don't think this suggestion would work anyway in the case of multiple redirects. Anyway, the existing implementation is good